mysql-test/include/percona_global_temp_table_alter.inc

```diff
@@ -0,0 +1,36 @@
+--source include/have_debug.inc
+--source include/have_debug_sync.inc
```

Comment:
May be move these 2 lines into the main .test file so that they would be invoked only once.